src/components/rockets/Reserve/Reserve.jsx

```diff
@@ -8,14 +8,23 @@ const Reserve = (props) => {
   const { id, reserved } = props;
   const dispatch = useDispatch();
 
-  const reserve = () => {
+  const toggleReservations = () => {
     dispatch(reserveRocket(id));
   };
 
   const text = reserved ? 'Cancel Reservation' : 'Reserve Rocket';
 
   return (
-    <button className="reserve-btn" onClick={reserve} type="button">
+    <button
+      className="reserve-btn"
+      onClick={toggleReservations}
+      type="button"
+      style={{
+        backgroundColor: reserved ? '#a3a3a310' : '#2897ff',
```

Comment:
Dear @Danie12345 please consider using classes when setting/switching several properties, it is easier to maintain and more readable.

i.e.
`className={ reserved ? 'reserved-class' : 'not-reserved' }`

Reply:
Thank you. I'll add it. 👍